datadog_checks_dev/tests/config_validator/test_config_block.py

```diff
@@ -0,0 +1,182 @@
+import os
```

Comment:
Any tooling tests need to go in https://github.com/DataDog/integrations-core/tree/master/datadog_checks_dev/tests/tooling

Reply:
👍